cf-agent/verify_packages.c

```diff
@@ -201,7 +201,7 @@ PromiseResult VerifyPackagesPromise(EvalContext *ctx, const Promise *pp)
             break;
         case PACKAGE_PROMISE_TYPE_OLD:
             Log(LOG_LEVEL_VERBOSE,
-                "Using old package promise. Please note that this will be obsolete in 3.8");
+                "Using old package promise. Please note that the old implementation is considered obsolete as of 3.8. The old implementation will continue to work, but forward development will be directed toward the new implementation.");
```

Comment:
OTOH, the number of unsupported package managers by the new package promise does make a convincing case to keep it around a bit longer. Maybe disregard my comment for now. We should make it clear once we have a deprecation path though.